engine/combat/utility/sos_consideration.py

```diff
@@ -49,6 +49,14 @@ def on_selected_character(
     def _default_appraisals(self: Self) -> list[SoSAppraisal]:
         """Generate default appraisals generic to every consideration."""
         attacks: list[BasicAttack] = []
+        if self.actor.character is PlayerPartyCharacter.Moraine:
+            moraine_rune = BasicAttack(
+                caster=self.actor.character,
+                boost=0,
+            )
+            attacks.append(moraine_rune)
```

Comment:
Did you test this? Hopefully it'll work, but might also do some weird stuff since Moraine's hold rune only needs a single input, not two (no targeting either).

Reply:
I am pretty sure it needed two inputs during yesterdays run. There was targeting but the only option was the dweller.